sumologic-k8s-api-cronjob-rbac.yaml

```diff
@@ -17,12 +17,17 @@ spec:
           restartPolicy: OnFailure
           containers:
           - name:  sumologic-k8s-api
-            image: frankreno/sumologic-k8s-api:latest
+            imagePullPolicy: Always
+            image: mcluskey/sumologic-k8s-api:events
```

Comment:
can we revert these to previous values? Once we get the script changes in i will update docker hub image.

Reply:
Whoops, this was an accident... I wasn't really thinking this would get merged.